js/assessments/checkForTooLongSentences.js

```diff
@@ -0,0 +1,17 @@
+var forEach = require( "lodash/forEach" );
+
+/**
+ * Counts number of too long sentences.
+ * @param {array} sentences The array containing sentences.
+ * @returns {number} Number of too long sentences.
+ */
+module.exports = function ( sentences ) {
```

Comment:
Consider moving this from assessments into a `/helpers/` folder and allow an optional parameter that will set the `recommendedValue` and use 20 as a fallback value.